packages/visual-editor/src/utils/schema/resolveSchema.ts

```
@@ -39,3 +48,139 @@ export const resolveSchemaJson = (
+            postalCode: child.address.postalCode,
+            addressCountry: child.address.countryCode,
+          },
+          phone: child.mainPhone,
+        },
+      };
+    });
+  }
+
+  switch (key) {
+    case "openingHours":
+      const hoursSchema = OpeningHoursSchema(resolvedValue);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)